src/libstd/collections/hash/set.rs

```diff
@@ -292,21 +292,21 @@ impl<T, S, H> HashSet<T, S>
     ///
     /// ```
     /// use std::collections::HashSet;
-    /// let a: HashSet<int> = [1, 2, 3].iter().map(|&x| x).collect();
-    /// let b: HashSet<int> = [4, 2, 3, 4].iter().map(|&x| x).collect();
+    /// let a: HashSet<int> = [1, 2, 3].iter().cloned().collect();
+    /// let b: HashSet<int> = [4, 2, 3, 4].iter().cloned().collect();
```

Comment:
This file is all a dupe of https://github.com/rust-lang/rust/pull/22242 as far as I can tell.